src/validation.cpp

```diff
@@ -556,12 +556,12 @@ bool MemPoolAccept::PreChecks(ATMPArgs& args, Workspace& ws)
     const uint256& hash = ws.m_hash;
 
     // Copy/alias what we need out of args
-    TxValidationState &state = args.m_state;
     const int64_t nAcceptTime = args.m_accept_time;
     const bool bypass_limits = args.m_bypass_limits;
     std::vector<COutPoint>& coins_to_uncache = args.m_coins_to_uncache;
 
     // Alias what we need out of ws
+    TxValidationState &state = ws.m_state;
```

Comment:
nit in 174cb5330af4b09f3a66974d3bae783ea43b190e: When touching the code it would be good it the correct style was used right away, instead of adding another commit afterward to fix it up once more. (This increases the `git blame` and `git log -S` depth)

Reply:
Mm, definitely - I will keep this in mind!